common/rfb/Hostname.h

```diff
@@ -31,6 +31,10 @@ namespace rfb {
     const char* hostEnd;
     const char* portStart;
 
+    // Trim leading whitespace
+    while(isspace(*hi))
+      hi++;
+
     if (hi == NULL)
       throw rdr::Exception("NULL host specified");
```

Comment:
This check needs to be first or your new code will crash.

Reply:
You're dead on, I apologize for missing that